web/style/products.scss

```diff
@@ -0,0 +1,130 @@
+@import '_variables.scss';
+@import 'common.scss';
+.category-list {

```

Comment:
Again don't write page specific CSS, write it module/component specific, follow SMACSS principles